exporter/common.go

```diff
@@ -153,6 +154,58 @@ func unique(slice []string) []string {
 	return list
 }
 
+func listCollectionsWithoutViews(ctx context.Context, client *mongo.Client) ([]string, error) {
+	dbs, err := databases(ctx, client, nil, nil)
+	if err != nil {
+		return nil, errors.Wrap(err, "cannot make the list of databases to list all collections")
+	}
+
+	var res []string
```

Comment:
I'd make it a map for better performance. Then we wouldn't need `isInArray` function.